spec/main.spec.js

```diff
@@ -11,21 +29,14 @@ describe('Opal Node Runtime', function () {
     });
 
     it('should preserve Function methods', function() {
-      const fundamentalObjects = [
-        Function,
-        Boolean,
-        Error,
-        Number,
-        Date,
-        String,
-        RegExp,
-        Array
-      ];
+
       for (let index in fundamentalObjects) {
         const fundamentalObject = fundamentalObjects[index];
         expect(fundamentalObject.call, `${fundamentalObject.name}.call should be a Function`).to.be.an.instanceof(Function);
         expect(fundamentalObject.apply, `${fundamentalObject.name}.apply should be a Function`).to.be.an.instanceof(Function);
         expect(fundamentalObject.bind, `${fundamentalObject.name}.bind should be a Function`).to.be.an.instanceof(Function);
+        expect(fundamentalObject.toString(), `${fundamentalObject.name}.toString should be native function`).to.be.equal(fundamentalToStringValues[index]);
+        expect(fundamentalObject.toString().match(/^\s*function (\w+)/), `${fundamentalObject.name}.toString should be native function`).to.be.not.null;
```

Comment:
We could use a more precise test 
```js
expect(fundamentalObject.toString()).to.equal(`function ${fundamentalObject.name}() { [native code] }`);
```

We make sure that `Error.toString()` equals `function Error() { [native code] }`

Reply:
You are right, i wrote a quick and dirty test 😄 . It's done 👍